src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    public Lottery() {
+        super();
+    }
+
+    public Lottery(int value, String color) {
+        super(value, color);
+    }
+
+    public Lottery getRandomBall() {
+        ColorSupplier rndColor = new ColorSupplier();
+        String randomColor = rndColor.getRandomColor().toString();
+
+        Random rndValue = new Random();
+        int randomValue = rndValue.nextInt(100);
```

Comment:
100 should be made a constant field (static final field with name in UPPER_CASE)
https://www.javatpoint.com/java-constant